### PR TITLE
[ko] Update outdated files in dev-1.23-ko.1 (M34-M37)

### PR DESCRIPTION
**Pull request by jihoon-seo**
- Related issue: #30845 (**dev-1.23-ko.1**)
- Related PR: #30851 (**dev-1.22-ko.4**)

> * [ ]  M34.  `content/en/docs/concepts/workloads/controllers/cron-jobs.md`  | 23(+S) 14(-)
> * [ ]  M35.  `content/en/docs/concepts/workloads/controllers/deployment.md`  | 25(+S) 9(-)
> * [ ]  M36.  `content/en/docs/concepts/workloads/controllers/job.md`  | 55(+M) 6(-)
> * [ ]  **M37.**  `content/en/docs/concepts/workloads/controllers/statefulset.md`  | **109(+L) 9(-)**

/language ko

### CODE REVIEW

**Review by yoonian — COMMENTED** [review]
@jihoon-seo 간단히 살펴보았습니다. 감사합니다. 👍

File: content/ko/docs/concepts/workloads/controllers/cron-jobs.md [hunk 1]
```diff
@@ -28,6 +26,16 @@ kube-controller-manager 컨테이너에 설정된 시간대는
 크론잡 컨트롤러가 사용하는 시간대로 결정한다.
 {{< /caution >}}
 
+{{< caution >}}
+[v1 CronJob API](/docs/reference/kubernetes-api/workload-resources/cron-job-v1/)은 
+위에서 설명한 타임존 설정을 공식적으로 지원하지는 않는다.
+
+`CRON_TZ` 또는 `TZ`와 같은 변수를 설정하는 것은 쿠버네티스 프로젝트에서 공식적으로 지원하지는 않는다.
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/controllers/cron-jobs.md:33]:
```suggestion
`CRON_TZ` 또는 `TZ` 와 같은 변수를 설정하는 것은 쿠버네티스 프로젝트에서 공식적으로 지원하지는 않는다.
```

File: content/ko/docs/concepts/workloads/controllers/cron-jobs.md [hunk 2]
```diff
@@ -28,6 +26,16 @@ kube-controller-manager 컨테이너에 설정된 시간대는
...
+`CRON_TZ` 또는 `TZ`와 같은 변수를 설정하는 것은
```
Comment (yoonian) [inline content/ko/docs/concepts/workloads/controllers/cron-jobs.md:34]:
```suggestion
`CRON_TZ` 또는 `TZ` 와 같은 변수를 설정하는 것은 
```

**Review by yoonian — COMMENTED** [review]
@jihoon-seo 눈에 보여서 추가했습니다.

**Comment by jihoon-seo** [comment]
@yoonian 리뷰 감사합니다. 리뷰 의견을 모두 반영하였습니다. 😊

**Pull request merged**